src/components/graph/graphService.ts

```diff
@@ -0,0 +1,60 @@
+import { Client } from '@microsoft/microsoft-graph-client';
+import { AuthCodeMSALBrowserAuthenticationProvider } from '@microsoft/microsoft-graph-client/authProviders/authCodeMsalBrowser';
+import { User } from 'microsoft-graph';
+import { adGroupsModel, adGroupModel } from './adGroupModel';
+
+let graphClient: Client | undefined = undefined;
+
+function ensureClient(authProvider: AuthCodeMSALBrowserAuthenticationProvider) {
```

Comment:
remember return type (when feasible)
```suggestion
function ensureClient(
  authProvider: AuthCodeMSALBrowserAuthenticationProvider
): Client {
```